### PR TITLE
Remove inline styles, remove `unsafe-inline` from style-src CSP, remove `x-powered-by` everywhere

### PR DESCRIPTION
**Pull request by davemcorwin**
## Changes proposed in this pull request:
- #3852 
- Remove inline styles
- remove `unsafe-inline` from `style-src` CSP
- remove `x-powered-by` header

## security considerations
addresses findings from ZAP, tightens up CSP and removes x-powered-by header

### CODE REVIEW

File: views/home.njk [hunk 1]
```diff
@@ -31,7 +31,7 @@
             </p>
           {% endif %}
           {% if authGithub %}
-            <div class="well-gray-lightest" style="text-align: center;">
+            <div class="well-gray-lightest txt-center">
```
Comment (hursey013) [inline views/home.njk:34]:
```suggestion
            <div class="well-gray-lightest text-center">
```

**Review by hursey013 — CHANGES_REQUESTED** [review]
One small tweak, otherwise looks good!

**Comment by pburkholder** [comment]
Is this up on dev/staging site where I can scan to ensure a clean result?

**Comment by davemcorwin** [comment]
@pburkholder after it's merged, it will be in staging.

**Review by hursey013 — APPROVED** [review]

**Pull request merged**